scripts/rosversion

```diff
@@ -37,7 +38,7 @@ import rospkg
 from rospkg.common import PACKAGE_FILE
 from rospkg.rospack import ManifestManager
 
-USAGE = "Usage: rosversion <package/stack> or rosversion -d"
+USAGE = """Returns the version of the given package, or the distribution version of ROS."""
```

Comment:
Since this is not a usage line anymore I have moved the string directly to the `ArgumentParser` constructor.